examples/plot_nii_to_graph.py

```diff
@@ -0,0 +1,201 @@
+"""
+.. _nii_to_graph:
+
+=========================================================
+Compute connecivity matrices and graph properties from nii files
+=========================================================
+The nii_to_graph pipeline performs graph analysis from functional MRI file
+in NIFTI format.
+
+The **input** data should be a preprocessed, and in the same space (e.g. MNI
+space) as the template used to define the nodes in the graph.
+"""
+
+# Authors: David Meunier <david_meunier_79@hotmail.fr>
+
+# License: BSD (3-clause)
+import os.path as op
+
+import nipype.pipeline.engine as pe
+
+from nipype.interfaces.utility import IdentityInterface
+import nipype.interfaces.io as nio
+
+import json  # noqa
+import pprint  # noqa
+
+from graphypype.utils_tests import load_test_data
+
+###############################################################################
+# Check if data are available
+
+import os
+
+data_path = load_test_data("data_nii")
+
+ROI_mask_file = op.join(data_path,"ROI_HCP","indexed_mask-ROI_HCP.nii")
+ROI_coords_file = op.join(data_path,"ROI_HCP","ROI_coords-ROI_HCP.txt")
+ROI_MNI_coords_file =op.join(data_path,"ROI_HCP","ROI_MNI_coords-ROI_HCP.txt")
+ROI_labels_file = op.join(data_path,"ROI_HCP","ROI_labels-ROI_HCP.txt")
+
+###############################################################################
+# Then, we create our workflow and specify the `base_dir` which tells
+# nipype the directory in which to store the outputs.
+
+# workflow directory within the `base_dir`
+conmat_analysis_name = 'conmat'
+
+from graphpype.pipelines.nii_to_conmat import create_pipeline_nii_to_conmat # noqa
```

Comment:
you should change the namespace. Should be able to do:

```py
from graphpype.pipelines import create_pipeline_nii_to_conmat
```

might need to update the __init__.py to make it work